packages/smart-contracts/scripts-create2/utils/chains.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+type ChainConfig = {
+  name: string;
+  chainId: number;
+  rpcUrls: string[];
+};
+
+const cachedChainConfigs: Record<string, ChainConfig> = {};
+
+export const getChainConfig = async (chainName: string) => {
```

Comment:
- moved this function out from the `toolbox` utils to the `smart-contracts` utils
- added a simple cache